src/clinical/service-worker-thread/tasks.ts

```diff
@@ -106,18 +106,21 @@ const sortDocs = (sortQuery: string, entityName: string, completionStats: Comple
 ) => {
   // Sort Value: 0 order is Unchanged, -1 Current lower index than Next, +1 Current higher index than Next
   let order = 0;
+  const isDescending = sortQuery.startsWith('-');
   const isDefaultSort =
-    entityName === ClinicalEntitySchemaNames.DONOR && sortQuery.includes('donorId');
-  const isDescending = sortQuery.startsWith('-') ? -1 : 1;
-  const key = isDescending === -1 ? sortQuery.split('-')[1] : sortQuery;
+    entityName === ClinicalEntitySchemaNames.DONOR &&
+    sortQuery.includes('completionStats.coreCompletionPercentage');
+
+  const queryKey = isDescending ? sortQuery.split('-')[1] : sortQuery;
+  const key = queryKey === 'donorId' ? 'donor_id' : queryKey;
```

Comment:
this looks good - wondering where the disconnect is between our naming conventions - something to keep an eye on

Reply:
As far as I can remember and from what I can see when reviewing the codebase, some of this is actually in response to requirements on the UI. This was an issue at the very beginning of building Submitted Data features and is something I was hoping to revisit one day.

Reply:
I suspected it was UI. Is it a case of enforcing consistency with how we name vars in our queries?

Reply:
There are Types (such as DonorCentricRecord) where it's `donor_id`, but our Donor model uses `donorId`. 
Because the early work on this feature was largely based off of existing code from other features, naming collisions may have started there. I could definitely clean this up a little with some dedicated time to review.